src/widgets/process_table/proc_widget_column.rs

```diff
@@ -78,6 +90,12 @@ impl ColumnHeader for ProcColumn {
             ProcColumn::State => "State",
             ProcColumn::User => "User",
             ProcColumn::Time => "Time",
+            #[cfg(feature = "gpu")]
+            ProcColumn::GpuMem => "GMEM",
+            #[cfg(feature = "gpu")]
+            ProcColumn::GpuMemPercent => "GMEM%",
```

Comment:
Nit for consistency:

```suggestion
            ProcColumn::GpuMem => "GMem",
            #[cfg(feature = "gpu")]
            ProcColumn::GpuMemPercent => "GMem%",
```